openstack/neutron/templates/vct-dvs-agent-configmap.yaml

```diff
@@ -33,7 +34,4 @@ template: |
       vsphere_password = {= password | quote =}
 
       network_maps = "{= physical | ini_escape =}:{= bridge | ini_escape =}"
-
-      [DVS]
-      portgroup_retention_iterations = 50
```

Comment:
You forgot to port these changes to the new file.

Reply:
Thx, i missed that